control_plane/attachment_service/src/service.rs

```diff
@@ -336,6 +342,32 @@ impl Service {
         tracing::info!("Startup complete, spawned {reconcile_tasks} reconciliation tasks ({shard_count} shards total)");
     }
 
+    async fn initial_heartbeat_round(&self) -> HashMap<NodeId, PageserverUtilization> {
+        assert!(!self.startup_complete.is_ready());
+
+        let nodes = {
+            let locked = self.inner.read().unwrap();
+            locked.nodes.clone()
+        };
+
+        let res = self.heartbeater_handler.heartbeat(nodes).await;
+
+        let mut online_nodes = HashMap::new();
+        if let Ok(deltas) = res {
+            for (node_id, status) in deltas.0 {
+                tracing::info!("HB: {node_id} -> {status:?}");
```

Comment:
nit: HB: "heartbeat" is reasonably terse, probably don't need to make it an acronym